src/dist/manifestation.rs

```diff
@@ -120,15 +120,15 @@ impl Manifestation {
         let installed_manifest_path = prefix.path().join(&rel_installed_manifest_path);
 
         // Create the lists of components needed for installation
-        let update = Update::build_update(self, new_manifest, changes, notify_handler)?;
+        let config = self.read_config()?;
+        let update = Update::build_update(self, new_manifest, &changes, &config, notify_handler)?;
 
         if update.nothing_changes() {
+            // TODO changes and ,manifest are empty?
+            eprintln!("update: {:?}", update);
```

Comment:
We need to make a decision about what's going on here

Reply:
I think this is just a stray TODO and println from my debugging and can be deleted

Reply:
unless we're hitting it in a test which is still failing

Reply:
Okay we're not hitting it so we can remove this.  Currently the test suite passes, though I expect that we need to add some more profile-specific tests.